airflow/models.py

```diff
@@ -2280,6 +2281,8 @@ def render_template_from_field(self, attr, content, context, jinja_env):
             result = {
                 k: rt("{}[{}]".format(attr, k), v, context)
                 for k, v in list(content.items())}
+        elif isinstance(content, Number):
+            result = content
```

Comment:
where does the render happened ?

Reply:
Should it be at L2305 TO l2307:
```
         if (
                 isinstance(content, six.string_types) and	                 isinstance(content, six.string_types) and
                 any([content.endswith(ext) for ext in exts])):	                 any([content.endswith(ext) for ext in exts])):
             return jinja_env.get_template(content).render(**context)	             return jinja_env.get_template(content).render(**context)
elif isinstance(content, Number):
   return jinja_env.get_template(content).render(**context)
```

I may be wrong, but just looking at the code not sure how the render happen, may be a unit test will help.

Reply:
thanks for the info. lgtm